### ISSUE TITLE
multi-line HTML text getting "Strings must use doublequote" error

### ISSUE THREAD

**Issue by mondaychen** [STRML/JSXHint#72]
My code:

````js
require(
  ["react-with-addons"],
  function (React) {
    "use strict";
    var getEmptyCollectionRender = function() {
      return (
        <div>
          Add member attributes
          to appear in here.
        </div>
      );
    };
});

````

My .jshintrc:
````
{
  "strict": true,
  "esnext": true,
  "laxbreak": true,
  "quotmark" : "double",
  "undef": true,
  "predef": ["window", "define", "require", "_", "$"]
}

````

I'm getting this error: Strings must use doublequote on the line `Add member attributes`, which is not a String.

The error is gone if I merge the two lines into one, like this:

````js
      return (
        <div>
          Add member attributes to appear in here.
        </div>
      );
````

However in real world the text could be too long to fit in only one line.

**Comment by mondaychen** [comment]
https://github.com/STRML/JSXHint/issues/21
https://github.com/facebook/react/pull/970

Seems that JSX complier causes this issue because it turns
````html
<div>
    line 1
    line 2
</div>
````
into
````js
React.createElement("div", null, 
    "line1" + ' ' +
    "line2"
);
````

So neither of the quotmark rules can work.

**Comment by STRML** [comment]
This can be fixed with #73. But in any case, use ESLint.
